spec/file-icons-spec.coffee

```diff
@@ -21,3 +25,45 @@ describe 'FileIcons', ->
     FileIcons.resetService()
 
     expect(FileIcons.getService()).not.toBe(service)
+
+  
+  describe 'Class handling', ->
```

Comment:
@as-cii I'm having an avalanche of issues trying to get these specs to run in `tree-view-spec.coffee` and I don't know why...